prow/test/integration/prow/cluster/100_namespace.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-pods
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: default
+  name: oauth-token
+data:
+  oauth: ZmFrZW9hdXRodG9rZW4K # From 'fakeoauthtoken'
+---
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Would it be possible to use the manifests from the config directory (maybe the starter-s3.yaml) to make sure they are correct?

Reply:
That would also be what I prefer, since majority of the manifests are identical. But there will be slight difference in deployment config, such as the image path, and future github-endpoint for github related integration tests, as well as other services that need mock(s). These can be achieved though, by various different method, such as kustomize, but might need some maintenance. What do you think?